### ISSUE TITLE
Bias results to NSW

### ISSUE THREAD

**Issue by JasonThomasData** [JasonThomasData/DocLink#15]
Failure to log - Wed, May 11.

Tried using the demo on Heroku. I found that when I typed in Green Valley, there was no predictive search.

![image](https://cloud.githubusercontent.com/assets/9067490/15168952/9fc58a60-1779-11e6-829c-02ffd03f9ba9.png)

The translation works, however.

![image](https://cloud.githubusercontent.com/assets/9067490/15168964/bccf9dbc-1779-11e6-8fc2-c7f3f91d07be.png)

But it returns all doctors no problem.

![image](https://cloud.githubusercontent.com/assets/9067490/15168976/e28a7b3a-1779-11e6-8eab-8a133b6a4bd6.png)

Same result in Firefox, IE 11.
I am using Windows 7.

**Comment by JasonThomasData** [comment]
Good news, I did a system reinstall and got PG working. The entire process is written in the Readme. However, the issue I described above is now happening on my local machine too. Must not be a Heroku issue.

**Comment by JasonThomasData** [comment]
Also, returning all doctors doesn't work locally at all.
Using Mint and Firefox

**Comment by JasonThomasData** [comment]
The reason why it failed on my local machine, is I didn't have any doctors on file. Why is that?
My doctors table is blank.

**Comment by equivalentideas** [comment]
> My doctors table is blank.

Have you added any doctors?

**Comment by JasonThomasData** [comment]
No, I'd thought that the database would be uploaded to Github when you
upload them.

**Comment by equivalentideas** [comment]
I think the issue here is that the search was for “Green Valley” and the geocoder didn't know to bias it to NSW. I think we should bias it for now since that's around area of focus.

The issue could be something else. @JasonThomasData if you can recreate the issue on the heroku version and post the link here that would help.

**Comment by equivalentideas** [comment]
> No, I'd thought that the database would be uploaded to Github when you
> upload them.

Nein, the database is separate from the app. I'll add something to the readme about how to add doctors.

**Comment by JasonThomasData** [comment]
Hi. yes it works now when I enter `Green Valley, NSW`. So yeah it looks like the bias option will work.
